README.md

```diff
@@ -1,2 +1,10 @@
 # number-partitioning
 Number partitioning problem coding exercise.
+
+The number partitioning problem begins with a set of numbers, S.  We must split the set of numbers into two sets with equal sum.  In our training course, we discuss this problem and how to create a QUBO to solve it.
+
+In this exercise, students are given an Ocean program outline and must create the QUBO dictionary (Q) that is provided to the D-Wave QPU and choose appropriate values for `chainstrength` and `numruns` on lines 24-25.
+
+To run your program type `python NPP.py`.  After the program has run, you will see output displaying the sums of the two sets, S0 and S1, returned by the QPU, followed by the list of numbers in set S1.  
```

Comment:
I recognize that `NPP.py` is a python script, and not a python package nor module. However, I just want to point out it seems fairly atypical to me to see a capitalized python filename. (i.e. consider using `npp.py`)
https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
fixed